### DISCUSSION TITLE
Enhancement: Score Outline/Shadow

### DISCUSSION THREAD

**Discussion by TaffySwordsman** [FunkinCrew/Funkin#1597] [Ideas]
In-game score text should have some sort of black outline/shadow to make it readable over any color and any background. Weeks 1-4 this isn't really an issue since they all have dark backgrounds but week 6 is okay at best and week 5 is flat out unreadable without really focusing on the text.

![image](https://user-images.githubusercontent.com/8838370/108578380-0a8e2900-72e8-11eb-8d4f-f750035b65b3.png)

**Comment by RagdollClash** [comment]
Duplicate of #228 and fixed with #336
